src/components/BaseDialog.tsx

```diff
@@ -21,6 +23,7 @@ const BaseDialog = ({
   secondaryActionLabel,
   onPrimaryAction,
   onSecondaryAction,
+  primaryActionDisabled = false,
```

Comment:
given how tailored this component is to the actions use case, consider renaming it `ActionsDialog`.

Reply:
Done